### ISSUE TITLE
JavaScript errors in Craft CMS CP when Article Editor plugin is enabled

### ISSUE THREAD

**Issue by robinsm** [creativeorange/craft-article#28]
**Description:**
When the Article Editor plugin is enabled, navigating between tabs in the Control Panel (CP) becomes impossible (in some entries but not all) due to JavaScript errors. Disabling the Article Editor plugin resolves the issue, indicating a potential conflict or bug within the plugin itself.

**Steps to Reproduce:**

- Ensure the Article Editor plugin is enabled.
- Navigate to any entry editing page in the CP.
- Ensure there is at least one instance of the Article Editor in the current entry.
- Attempt to switch tabs within the interface.

**Expected Behavior:**
The CP Tab navigation should work seamlessly without any errors, allowing users to edit and manage entries as needed.

**Actual Behavior:**
The console logs a JavaScript error related to a TypeError: Cannot read properties of undefined (reading '$tabContainer'), preventing tab navigation. This error seems to originate from the plugin's JavaScript files.

- disabling the Article Editor 'block' doesn't resolve the issue.
- deleting the Article Editor 'block' from the entry resolves the issue.

**Console Errors:**

`jQuery.Deferred exception: Cannot read properties of undefined (reading '$tabContainer') TypeError: Cannot read properties of undefined (reading '$tabContainer')`

```
Uncaught TypeError: Cannot read properties of undefined (reading '$tabContainer')
    at s.constructor.init (ElementEditor.js:94:44)
    at s.constructor.constructor (Base.js:20:15)
```

**Environment:**
Craft CMS version: 4.8.3 (and 4.8.6)
Article Editor Plugin version: 3.1.0

**Comment by Edsardio** [comment]
Hi @robinsm 

I seem to be unable to reproduce the issue. Are you able to create a repository I can clone to reproduce the issue?

**Comment by krisMoca** [comment]
Hey I'm getting the same issue. 
Craft CMS Solo version: 4.9.6
Article Editor Plugin version: 3.1.0

![image](https://github.com/creativeorange/craft-article/assets/86084360/665b7381-3a03-47ff-a4cb-2c935d14acf2)

**Comment by MatthewBUK** [comment]
Hi @Edsardio ,

I seem to be getting the same issue after upgrading to latest version of Craft 4.
![Screenshot 2024-07-18 at 09 14 17](https://github.com/user-attachments/assets/af92b522-2a45-4281-b3c9-5ee251801bc0)

[Update]
I rolled back all my updated to Craft and Article Editor back to 2.0.1 and 4.5.5 then ran just the Craft updates and its working as expected. I have noted by reverting EditorAssets.php back to `use yii\web\JqueryAsset;`  and 

` $this->depends = [
            JqueryAsset::class,
        ];
`
Its working again. Might need to relook at this issue in Craft 4 and then again in Craft 5.

This fix works for Craft 5.2.8
